Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -783,6 +813,56 @@ theorem comap_incl_eq_bot : N₂.comap N.incl = ⊥ ↔ N ⊓ N₂ = ⊥ := by
     disjoint_iff, inf_coe_toSubmodule]
 #align lie_submodule.comap_incl_eq_bot LieSubmodule.comap_incl_eq_bot
 
+@[mono]
+theorem map_mono (h : N ≤ N₂) : N.map f ≤ N₂.map f :=
+  Set.image_subset _ h
+
+theorem map_comp
+    {M'' : Type*} [AddCommGroup M''] [Module R M''] [LieRingModule L M''] {g : M' →ₗ⁅R,L⁆ M''} :
+    N.map (g.comp f) = (N.map f).map g :=
+  SetLike.coe_injective <| by
+    simp only [← Set.image_comp, coe_map, LinearMap.coe_comp, LieModuleHom.coe_comp]
+
+@[simp]
+theorem map_id : N.map LieModuleHom.id = N := by ext; simp
+
+@[simp] theorem map_bot :
+    (⊥ : LieSubmodule R L M).map f = ⊥ := by
+  ext m; simp [eq_comm]
+
+lemma map_rel_iff (hf : Function.Injective f) :
```

Comment:
This name looks a bit weird to me. Does it match the Submodule name?

Reply:
Thanks, renamed to `map_le_map_iff`.